xApps/python/lib/e2sm_rc_module.py

```diff
@@ -56,15 +56,15 @@ def send_control_request_style_2_action_6(self, e2_node_id, ue_id, min_prb_ratio
                                     {'ranParameter-ID': 1, 'ranParameter-valueType': ('ranP-Choice-List', {'ranParameter-List': {'list-of-ranParameter': [{'sequence-of-ranParameters': [
                                         {'ranParameter-ID': 3, 'ranParameter-valueType': ('ranP-Choice-Structure', {'ranParameter-Structure': {'sequence-of-ranParameters': [
                                             {'ranParameter-ID': 4, 'ranParameter-valueType': ('ranP-Choice-List', {'ranParameter-List': {'list-of-ranParameter': [{'sequence-of-ranParameters': [
```

Comment:
Shouldn't this param `RRM Policy Member List` ID be `ranParameter-ID: 5` (currently it's `4`) based on the specifications?
<details>
<summary>Specifications</summary>
<img width="1029" alt="image" src="https://github.com/user-attachments/assets/79c00570-a7fa-45a6-9966-f714a2b0f5fc">
</details>

Also, sorry if this is a silly question, but I'm wondering why does control message ignore the first `STRUCTURE` type elements of the list, such as `RRM Policy Ratio Group - ID 2` and `RRM Policy Member - ID 6` RAN parameters?

Reply:
@pgawlowicz

Reply:
indeed, sth is wrong, i will investigate it. Thanks for pointing out.